tests/base.py

```diff
@@ -595,6 +595,254 @@ def setUp(self, config_dict='config_dict'):
+              'endOfRange': 10000
+            }
+          ],
+          'audienceIds': ['3468206642', '3988293898', '3988293899', '3468206646',
+                          '3468206647', '3468206644', '3468206643'],
+          'variations': [
+            {
+              'variables': [
+                {
+                  'id': '11535264366',
+                  'value': 'xyz'
+                }
+              ],
+              'id': '11617170975',
+              'key': 'variation_2',
+              'featureEnabled': True
+            }
+          ],
+          'forcedVariations': {},
+          'id': '11564051718'
+        },
+        {
+          'id': '1323241597',
+          'key': 'typed_audience_experiment',
+          'layerId': '1630555627',
+          'status': 'Running',
+          'variations': [
+            {
+              'id': '1423767503',
+              'key': 'A',
+              'variables': []
+            }
+          ],
+          'trafficAllocation': [
+            {
+              'entityId': '1423767503',
+              'endOfRange': 10000
+            }
+          ],
+          'audienceIds': ['3468206642', '3988293898', '3988293899', '3468206646',
+                          '3468206647', '3468206644', '3468206643'],
+          'forcedVariations': {}
+        }
+      ],
+      'audiences': [
+        {
+          'id': '3468206642',
+          'name': 'exactString',
+          'conditions': '["and", ["or", ["or", {"name": "house", "type": "custom_attribute", "value": "Gryffindor"}]]]'
+        },
+        {
+          'id': '3988293898',
+          'name': '$$dummySubstringString',
+          'conditions': '{ "type": "custom_attribute", "name": "$opt_dummy_attribute", "value": "impossible_value" }'
+        },
+        {
+          'id': '3988293899',
+          'name': '$$dummyExists',
+          'conditions': '{ "type": "custom_attribute", "name": "$opt_dummy_attribute", "value": "impossible_value" }'
+        },
+        {
+          'id': '3468206646',
+          'name': '$$dummyExactNumber',
+          'conditions': '{ "type": "custom_attribute", "name": "$opt_dummy_attribute", "value": "impossible_value" }'
+        },
+        {
+          'id': '3468206647',
+          'name': '$$dummyGtNumber',
+          'conditions': '{ "type": "custom_attribute", "name": "$opt_dummy_attribute", "value": "impossible_value" }'
+        },
+        {
+          'id': '3468206644',
+          'name': '$$dummyLtNumber',
+          'conditions': '{ "type": "custom_attribute", "name": "$opt_dummy_attribute", "value": "impossible_value" }'
+        },
+        {
+          'id': '3468206643',
+          'name': '$$dummyExactBoolean',
+          'conditions': '{ "type": "custom_attribute", "name": "$opt_dummy_attribute", "value": "impossible_value" }'
+        }
+      ],
+      'typedAudiences': [
+        {
+          'id': '3988293898',
+          'name': 'substringString',
+          'conditions': '["and", ["or", ["or", '
```

Comment:
After recent discussions, `conditions` will not be stringified but instead object (or list of objects and boolean operators)

Reply:
@nchilada ^

Reply:
I see. Let me revise it.